bank-balance-exactly-once/src/main/java/com/github/simplesteph/udemy/kafka/streams/BankBalanceExactlyOnceApp.java

```diff
@@ -53,22 +47,21 @@ public static void main(String[] args) {
         initialBalance.put("time", Instant.ofEpochMilli(0L).toString());
 
         KTable<String, JsonNode> bankBalance = bankTransactions
-                .groupByKey(Serdes.String(), jsonSerde)
+                .groupByKey(Serialized.with(Serdes.serdeFrom(String.class),Serdes.serdeFrom(JsonNode.class)))
```

Comment:
Why can't you use `Serdes.String()`? 
Again, externalise `jsonSerde` like we had before and keep the code. only `Serialized.with` is added